infrastructure/terraform/bin/bootstrap_deployment_superuser_role.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -euo pipefail  # safe scripting
```

Comment:
do we need this script if Aiden's internal-repo change adds this as a terraform resource in there in bootstrap, happy to have that till its setup, as I think like web-gateway and templates repos we might have a couple things to refactor once those changes go in and we start doing it that way

Reply:
exactly yeah - I'm just trying to make sure we've at least got an equivalent to the web-gateway stuff.

Reply:
agreed, this will just be an element of tech debt until a solution is found and they can be brought in to line